docker/local-dev-bsp-compose.yml

```diff
@@ -23,6 +23,7 @@ services:
         "--keystore-path=/keystore",
         "--sealing=manual",
         "--base-path=/data",
+        "--log=runtime=debug",
```

Comment:
Maybe we shouldn't keep this in main. I'd guess you used it for debugging, and was left here.